boards/ct/ctcc/ctcc_nrf52840.dts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2024 CTHINGS.CO
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "CTHINGS.CO Connectivity Card nRF52840";
+	compatible = "ct,ctcc-nrf52840";
+
+	chosen {
+		zephyr,console = &cdc_acm_uart;
+		zephyr,shell-uart = &cdc_acm_uart;
+		zephyr,uart-mcumgr = &cdc_acm_uart;
+		zephyr,bt-mon-uart = &cdc_acm_uart;
+		zephyr,bt-c2h-uart = &cdc_acm_uart;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,ieee802154 = &ieee802154;
+	};
+
+	leds {
+		 compatible = "gpio-leds";
+		 led1: led_1 {
+			 gpios = <&gpio0 23 GPIO_ACTIVE_LOW>;
+			 label = "LED 1";
+		 };
+		 led2: led_2 {
+			 gpios = <&gpio0 22 GPIO_ACTIVE_LOW>;
+			 label = "LED 2";
+		 };
+	};
+
+	aliases {
+		led0 = &led1;
+		led1 = &led2;
+		mcuboot-led0 = &led1;
+	};
+};
+
+&flash0 {
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00012000>;
+		};
+		slot0_partition: partition@12000 {
+			label = "image-0";
+			reg = <0x00012000 0x00075000>;
+		};
+		slot1_partition: partition@87000 {
+			label = "image-1";
+			reg = <0x00087000 0x00075000>;
+		};
```

Comment:
not really sure what went on here?
Reduce boot by 0x2000 and make slot0 0x77000, keep slot1 at 0x75000

Reply:
I don't understand, how this will make enabled dbg logs from MCUboot fit into `mcuboot` partition? Having mcuboot partition with size `0x10000` and enabled logs `CONFIG_LOG_DEFAULT_LEVEL=4`  `CONFIG_MCUBOOT_LOG_LEVEL_DBG=y` `CONFIG_LOG_OVERRIDE_LEVEL=4` will results with `region `FLASH' overflowed by 6308 bytes`. Also, shouldn't images have the same size? Or am I misunderstanding something?

Reply:
Ah, if your MCUboot needs to be bigger then the increased size for that is fine. MCUboot's overhead sizes depend upon what mode you use, if you use swap using move (the default) which allows for image reverts then you need 1 sector to shift the image up by and an additional sector for the swap status data, this is why slot0 should be 2 sectors larger than slot1. If you pick these sizes, the upgrade will work, but slot1 will have 2 sectors you can never use, so you could take them off and put them in storage or take one from slot1 and add it to slot0

Reply:
Thanks for the explanation! I added 2 sectors to slot0, so now everything should be fine.